### PR TITLE
 Document xrootd-lcmaps usage with LCMAPS VOMS (SOFTWARE-2949) 

### PR DESCRIPTION
**Pull request by brianhlin**
@zvada could you also take a look at this?

@matyasselmeci please run the instructions to set up a standalone xrootd server with xrootd-lcmaps

### CODE REVIEW

File: docs/security/host-certs.md [hunk 1]
```diff
@@ -143,38 +143,52 @@ root@host # chmod 400 /etc/grid-security/hostkey.pem
...
+
+1. Set the ownership of the directory and its underlying files to the Unix user, indicated as `<USER>`, who runs the service:
+
+        :::console
+        root@host # chown -R %RED%<USER>%ENDCOLOR%.%RED%<USER>%ENDCOLOR% /etc/grid-security/%RED%<SERVICE>%ENDCOLOR%/
+
+    For example, both the Apache and Tomcat services are run by the `tomcat` user:
+
+        :::console
+        root@host # chown -R tomcat.tomcat /etc/grid-security/http/
```
Comment (matyasselmeci) [inline docs/security/host-certs.md]: `tomcat.tomcat` is outdated syntax; should be `tomcat:tomcat`.

File: docs/data/install-xrootd.md [hunk 2]
```diff
@@ -514,128 +514,107 @@ perform several steps to make xrootdfs secured.
...
+        Run: [FATAL] Auth failed
 
-On the `xrootd-client` node,
+1. On the XRootD host, add your DN to [/etc/grid-security/grid-mapfile](/security/lcmaps-voms-authentication#mapping-users)
 
-``` console
-user@host $ xrdcp /bin/bash root://fermicloud121.fnal.gov//tmp/lcmaps_test
-[xrootd] Total 0.73 MB  [====================] 100.00 % [inf MB/s]
-```
+1. Generate your proxy and verify that you can successfully transfer files:
```
[the remark this answers is not in the document]
Comment (matyasselmeci) [reply docs/data/install-xrootd.md]: ... I'm not sure if you have to edit the `auth_file` for both the redirector and all the data nodes, or just the data nodes. I'll update if I find out.

[the remark this answers is not in the document]
Comment (matyasselmeci) [reply docs/data/install-xrootd.md]: also, say something about using `--debug 3` to do troubleshooting.

[the remark this answers is not in the document]
Comment (zvada) [reply docs/data/install-xrootd.md]: `-d 2` is more readable and if obvious issue is present it'll show it. I'd not recommend `-d 3` unless we really have a reason for it.

[the remark this answers is not in the document]
Comment (brianhlin) [reply docs/data/install-xrootd.md]: @matyasselmeci did you ever find it if you have to update `auth_file`s on all hosts or just the redirector?

**Comment by brianhlin** [comment]
@matyasselmeci ready for re-review

File: docs/data/install-xrootd.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (matyasselmeci) [reply docs/data/install-xrootd.md]: I think it's just on the data nodes. (Not the redirector.)

[the remark this answers is not in the document]
Comment (matyasselmeci) [reply docs/data/install-xrootd.md]: Verified. Just the data nodes.

**Pull request merged**